adam/curriculum/curriculum_utils.py

```diff
@@ -163,6 +163,6 @@ def learner_template_factory() -> TemplateObjectVariable:
 def shuffle_curriculum(
     curriculum: List[Phase1InstanceGroup], *, rng: Random
 ) -> Sequence[Phase1InstanceGroup]:
-    curriculum = flatten(curriculum)
-    rng.shuffle(curriculum)
-    return curriculum
+    mod_curriculum = flatten(curriculum)
```

Comment:
Does this need to be `list(flatten(...))` to be `shuffle`-able? (Probably doesn't matter for the moment -- do we use this shuffle function anywhere currently?)
```suggestion
    mod_curriculum = list(flatten(curriculum))
```